doc/rados/operations/placement-groups.rst

```diff
@@ -135,28 +138,27 @@ example, a pool that maps to OSDs of class `ssd` and a pool that maps
 to OSDs of class `hdd` will each have optimal PG counts that depend on
 the number of those respective device types.
 
-The autoscaler uses the `scale-up` profile by default,
-where it starts out each pool with minimal PGs and scales
-up PGs when there is more usage in each pool. However, it also has
-a `scale-down` profile, where each pool starts out with a full complements 
-of PGs and only scales down when the usage ratio across the pools is not even.
+The autoscaler uses the `bulk` flag to determine which pool
+should start out with a full complements of PGs and only
+scales down when the the usage ratio across the pool is not even. 
+However, if the pool doesn't have the `bulk` flag, the pool will
+start out with minimal PGs and only when there is more usage in the pool.
 
-With only the `scale-down` profile, the autoscaler identifies
-any overlapping roots and prevents the pools with such roots
-from scaling because overlapping roots can cause problems
+The autoscaler identifies any overlapping roots and prevents the pools 
+with such roots from scaling because overlapping roots can cause problems
 with the scaling process.
 
-To use the `scale-down` profile::
+To create pool with `bulk` flag::
 
-  ceph osd pool set autoscale-profile scale-down
+  ceph osd create <pool-name> --bulk
 
-To switch back to the default `scale-up` profile::
+To set/unset `bulk` flag of existing pool::
 
-  ceph osd pool set autoscale-profile scale-up
+  ceph osd pool set <pool-name> bulk=true/false/1/0
 
-Existing clusters will continue to use the `scale-up` profile.
-To use the `scale-down` profile, users will need to set autoscale-profile `scale-down`,
-after upgrading to a version of Ceph that provides the `scale-down` feature.
+To get `bulk` flag of existing pool::
```

Comment:
"To query the `bulk` flag setting of an existing pool::"

Reply:
@anthonyeleven could you please incorporate your suggestions in a doc PR against master, which can then be backported? In most cases, backport PRs just include cherry-picked commits from master.

Reply:
That seems unusual, since my suggestions are relative to text that is new in this PR.  If you prefer I can wait for this PR to be merged then submit my suggestions in a followup PR.

Reply:
Incorporating the suggested changes directly in this Pacific PR would mean that they won't land in the master (or Quincy). You don't need to wait for this PR to merge, just open a PR against master whenever you like. We can cherry-pick the commit to relevant branches, in another followup PR or this one, based on when it lands.

Reply:
Oh, I missed that this was against the pacific branch.